pom.xml

```
@@ -43,8 +43,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.414.x</artifactId>
-        <version>2791.v707dc5a_1626d</version>
+        <artifactId>bom-2.462.x</artifactId>
```

Same reasoning as previous comment https://github.com/jenkinsci/gitlab-branch-source-plugin/pull/438/files#r1733243242
```suggestion
        <artifactId>bom-2.440.x</artifactId>
```